templates/default/contentlayer.config.js

```diff
@@ -63,6 +64,7 @@ export default makeSource({
   mdx: {
     remarkPlugins: [
       remarkGfm,
+      [smartypants, {quotes: false, dashes: 'oldschool'}],
```

Comment:
Let's also replace the quotes, they look a bit better this way :)

```suggestion
      [smartypants, {dashes: 'oldschool'}],
```